website/content/partials/components/builder-docker.mdx

```diff
@@ -86,6 +86,15 @@ Set this when the Dockerfile is not APP-PATH/Dockerfile.
 - Type: **string**
 - **Optional**
 
+#### platform
+
+Set platform if server is multi-platform capable.
+
+Recommended usage with buildkit enabled.
```

Comment:
Can you re-run `waypoint gen/website-mdx` now that you've updated the doc string?

Reply:
* `make gen/website-mdx`
Also proposal for mountain bikes to pop up in the terminal when typo-ing website-dmx